db/migrate/20220701104355_create_article_categories.rb

```diff
@@ -0,0 +1,8 @@
+class CreateArticleCategories < ActiveRecord::Migration[6.0]
+  def change
+    create_table :article_categories do |t|
```

Comment:
@ShivaniKandukuri here itself, we should add the relationship (references) instead of the `t.integer :article_id` and `t.integer :category_id`.